apps/web/src/components/Menu/hooks/useTradingRewardStatus.ts

```diff
@@ -0,0 +1,20 @@
+import { useMemo } from 'react'
+import useAllTradingRewardPair from 'views/TradingReward/hooks/useAllTradingRewardPair'
+
+export const useTradingRewardStatus = () => {
+  const { data: allTradingRewardPairData } = useAllTradingRewardPair()
+  const latestCampaignId = allTradingRewardPairData.campaignIds[allTradingRewardPairData.campaignIds.length - 1]
```

Comment:
The `latestCampaignId` might be undefined?

Reply:
no, will always have one

Reply:
Yeah but it depends on the business logic right? If we just look at the code it could be undefined